srcpkgs/gf2x/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'gf2x'
+pkgname=gf2x
+version=1.3.0
+revision=1
+wrksrc=gf2x-gf2x-${version}
+build_style="gnu-configure"
+hostmakedepends="automake libtool"
+short_desc="C/C++ library for fast arithmetic in GF(2)[x]"
+maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.inria.fr/gf2x/gf2x"
+distfiles="https://gitlab.inria.fr/gf2x/gf2x/-/archive/gf2x-${version}/gf2x-gf2x-${version}.tar.gz"
+checksum=11bcf98b620c60c2ee3b4460b02b7be741f14cfdc26b542f22c92950926575e0
+
+build_options="native"
+
+if [ ! "$build_option_native" ]; then
+	configure_args="--disable-sse3"
+fi
+
+pre_configure() {
+	autoreconf -fi
+}
+
+gf2x-devel_package() {
+	depends="gf2x-${version}_${revision}"
```

Comment:
```suggestion
	depends="gf2x>=${version}_${revision}"
```

Reply:
I remembered it just now, but the default style is actually `${sourcepkg}>=...`

Reply:
done for all 5 templates